pages/knowyourrights/index.tsx

```diff
@@ -199,15 +199,23 @@ const KnowYourRights = () => {
                   <HighlightOffIcon className={classes.icons} />
                 </button>
                 {activeItem && (
-                  <img
+                  <a
                     style={{ marginTop: '2rem' }}
-                    id="simple-modal-title"
-                    src={
+                    href={
                       language === ENGLISH
                         ? `${activeItem.flyerPDF}`
                         : `${activeItem.spanishFlyerPDF} `
                     }
-                  />
+                    download="Flyer"
+                  >
+                    <img
+                      src={
+                        language === ENGLISH
```

Comment:
![Screen Shot 2021-03-22 at 3 22 23 PM](https://user-images.githubusercontent.com/72828520/112065744-6e438600-8b22-11eb-8c85-d4fce0fa5978.png)

On mobile views, I think it would make more sense to keep this centered.